package.json

```
@@ -2,41 +2,69 @@
     "name": "parse5-build-scripts",
     "private": true,
     "type": "module",
+    "workspaces": [
+        "packages/*",
+        "bench"
+    ],
     "devDependencies": {
+        "@types/dedent": "^0.7.0",
+        "@types/jest": "^27.4.0",
+        "@typescript-eslint/eslint-plugin": "^5.9.0",
+        "@typescript-eslint/parser": "^5.9.0",
         "dedent": "^0.7.0",
-        "eslint": "^7.30.0",
+        "eslint": "^8.6.0",
         "eslint-config-prettier": "^8.3.0",
-        "eslint-plugin-prettier": "^3.4.0",
         "eslint-plugin-unicorn": "^40.0.0",
-        "husky": "^0.14.3",
+        "husky": "^7.0.4",
+        "jest": "^27.4.7",
         "lerna": "^4.0.0",
-        "lint-staged": "^7.2.0",
-        "mocha": "^9.0.2",
-        "prettier": "^2.3.2",
-        "r2": "^2.0.1",
-        "typescript": "^4.3.5"
+        "lint-staged": "^11.2.6",
+        "prettier": "^2.5.0",
+        "ts-jest": "^27.1.2",
+        "ts-node": "^10.4.0",
+        "typescript": "^4.5.4"
     },
     "scripts": {
-        "build": "lerna run build",
-        "format": "prettier --write \"{packages/*/{lib,test}/**/*.js,{bench,test,scripts}/**/*.js}\"",
-        "bootstrap": "lerna bootstrap --no-ci",
-        "lint": "eslint --ext js .",
-        "unit-tests": "mocha --ui tdd --reporter progress --timeout 20000 packages/**/test/*.test.js",
+        "build": "npm run build --workspaces --if-present",
+        "prettier": "prettier '**/*.{js,ts,md,json,yml}'",
```

```suggestion
        "prettier": "prettier '**/*.{js,ts,md,json,yml}' --loglevel warn",
```

This prevents it from listing all the files it touches (most tools such as eslint don’t print that either)

---

Didn't know about this, thanks for flagging!